config/default/translations/messages-en.properties

```
@@ -131,4 +131,428 @@ task.pnc.danger_sign_followup_mother.title = PNC danger sign follow up - mother
+report.pregnancy_home_visit.update_g_age.previous_g_age = Weeks pregnant
+report.pregnancy_home_visit.update_g_age.previous_g_age_unknown = Unknown weeks pregnant
+report.pregnancy_home_visit.update_g_age.previous_edd = EDD: 
+report.pregnancy_home_visit.update_g_age.previous_edd_unknown = EDD: unknown
+report.pregnancy_home_visit.update_g_age.new_info = New
+report.pregnancy_home_visit.update_g_age.new_g_age = Weeks pregnant
+report.pregnancy_home_visit.update_g_age.new_edd = EDD: 
+report.pregnancy_home_visit.update_g_age.lmp_date_8601_new = LMPD DATE 8601 NEW 
+report.pregnancy_home_visit.update_g_age.lmp_date_new = LMP DATE
+report.pregnancy_home_visit.update_g_age.edd_8601_new = NEW EDD 8601
+report.pregnancy_home_visit.update_g_age.edd_new = NEW EDD
+
+report.pregnancy_home_visit.anc_visits_hf = Health facility ANC visits
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past = ANC Visits at Health Facility (Past)
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.pregnancy_follow_up_date_recent = Recent Follow up date
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.last_visit_attended = Woman completed the health facility ANC visit as scheduled
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.report_other_visits = Report any additional unreported health facility ANC visits
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_hf_count =  Number of additional unreported health facility ANC visits
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_date_ask_single = Facility Visit date 
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_date_single = Date
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates = Vist Dates
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates_count = Number of visit Dates
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.0 = First Visit Date
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.0.visited_date_ask_multiple = Visit
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.0.visited_date = Date
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.1.visited_date_ask_multiple = Visit
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.1.visited_date = Date
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.1 = Second Visit Date
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.2 = Third Visit Date
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.2.visited_date_ask_multiple = Visit
+report.pregnancy_home_visit.anc_visits_hf.anc_visits_hf_past.visited_dates.2.visited_date = Date
+
+report.pregnancy_home_visit.anc_visits_hf.risk_factors = Risk Factors
+report.pregnancy_home_visit.anc_visits_hf.risk_factors.previous_risk_factors_note = Risk factors:
```

We should remove the colon, unless that is consistently the notation for all header labels.

```suggestion
report.pregnancy_home_visit.anc_visits_hf.risk_factors.previous_risk_factors_note = Risk factors
```